src/components/hoverable-visuals.js

```diff
@@ -51,13 +51,25 @@ AFRAME.registerComponent("hoverable-visuals", {
 
```

Comment:
Make sure to also check the batched objects hover visuals in `HubsBatchRawUniformGroup`

Reply:
:+1: